### ISSUE TITLE
App starts logged out

### ISSUE THREAD

**Issue by asorel1942** [NicoVarg99/MEGA-Instances#10]
Upon restarting the machine (Manjaro KDE 64 bit), the two instances of MEGAsync I had set up to launch using the script launch logged out, rather than already logged into their respective accounts.

**Comment by H0R1Z0N** [comment]
I just noticed mine are also not running... welp...

**Comment by H0R1Z0N** [comment]
Is the dev still alive ?

**Comment by NicoVarg99** [comment]
Please provide the steps to reproduce the problem

**Comment by asorel1942** [comment]
Steps to reproduce were to create two MEGA instances for two different accounts using the script, then to log off or shut down the computer. Upon starting a new session, the two instances launched, but necessitated manual login for both.

**Comment by H0R1Z0N** [comment]
Yeah, i kinda did what was told to delete it from the file and it give "megasync: error while loading shared libraries: libraw.so.19: cannot open shared object file: No such file or directory"

**Comment by H0R1Z0N** [comment]
Trying to reinstall libraw, it seem arch moved to libraw 0.20.0-1

**Comment by H0R1Z0N** [comment]
I will try to see if i can rebuild the script and say 19 and higher

**Comment by NicoVarg99** [comment]
Does MEGA used without this script start logged in?

**Comment by asorel1942** [comment]
Yes, it does. Both before the script was implemented and after disabling it, the regular instance of MEGA behaves normally.

**Comment by bnsave100** [comment]
any chances for windows release ???

**Comment by vitto62** [comment]
Innanzitutto grazie per il tuo lavoro e la tua generosa condivisione!!!
**L'utilità del MEGA-Instances è immensa!** Io possiedo 7 account e da anni effettuo un context-switch a mano!!! Un vero sca..o.  :)


Perdonami Nicola ma il tuo inglese non consente di capire le tue indicazioni.  Ad esempio  "Does MEGA used without this script start logged in?" è quasi un nonsense. O perlomeno è fortemente ambiguo.  
Gli anglofoni sono troppo "polite" per sollevare questioni :)    Questa mia osservazione è solo per campanilismo Italiano: non facciamoci ricordare quanto siamo ignoranti in questo moderno volapük !  :)

A latere: non trovo indicazioni su come dovrebbe funzionare in run-time. L'immagine del system tray suggerisce che lancia "n" demoni MEGA in parallelo ma a me ne lancia solo uno. Oviamente l'ho configurato con istanze multiple rispondendo ai pop-up del primo lancio.

Ora provo a fare un reboot. Chissà se funziona ?
